README.md

```diff
@@ -1,10 +1,134 @@
+    - [27. Time series (TBD)](https://open-academy.github.io/machine-learning/deep-learning/time-series.html)
+    - [28. DQN (TBD)](https://open-academy.github.io/machine-learning/deep-learning/DQN.html)
+    - [29. Summary of deep learning (TBD)](https://open-academy.github.io/machine-learning/deep-learning/dl-summary.html)
+    - [30. Image classification](https://open-academy.github.io/machine-learning/deep-learning/image-classification.html)
+6. **MACHINE LEARNING PRODUCTIONIZATION**
+    - [31. Overview](https://open-academy.github.io/machine-learning/machine-learning-productionization/overview.html)
+    - [32. Problem framing](https://open-academy.github.io/machine-learning/machine-learning-productionization/problem-framing.html)
+    - [33. Data engineering](https://open-academy.github.io/machine-learning/machine-learning-productionization/data-engineering.html)
+    - [34. Model training & evaluation](https://open-academy.github.io/machine-learning/machine-learning-productionization/model-training-and-evaluation.html)
+    - [35. Model deployment](https://open-academy.github.io/machine-learning/machine-learning-productionization/model-deployment.html)
+7. **SUPPORTING MATERIALS**
+    - [36. PyTorch](https://open-academy.github.io/machine-learning/supporting-materials/pytorch.html)
+    - [37. Bamboolib](https://open-academy.github.io/machine-learning/supporting-materials/bamboolib.html)
+    - [38. Mito](https://open-academy.github.io/machine-learning/supporting-materials/mito.html)
+    - [39. KNN (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/KNN.html)
+    - [40. Semi-supervised learning (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/semi-supervised-learning.html)
+    - [41. Unbalanced problems](https://open-academy.github.io/machine-learning/supporting-materials/unbalanced-problems.html)
+    - [42. AutoML (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/automl.html)
+8. **OTHERS**
+    - [43. Assignments](https://open-academy.github.io/machine-learning/assignments/introduction.html)
+    - [44. Slides](https://open-academy.github.io/machine-learning/slides/introduction.html)
+
+### Curriculum
+
+We are committed to delivering high-quality and practical courses on schedule every week, featuring instructors from both academia and industry, including technical experts from leading technology companies. Currently, we have completed the section on Data Science and are now in progress with the Machine Learning section. Due to limited capacity and venue availability, the course will be conducted online via [online textbook](https://open-academy.github.io/machine-learning/intro.html) and [VooV (Tencent) meeting](https://www.tencentcloud.com/products/tvm), and at this time, we are only accepting students who are proficient in Chinese (although we plan to address this issue in the future). Please find the class schedule below.

```

Comment:
remove such content,

```
Due to limited capacity and venue availability, the course will be conducted online via [online textbook](https://open-academy.github.io/machine-learning/intro.html) and [VooV (Tencent) meeting](https://www.tencentcloud.com/products/tvm), and at this time, we are only accepting students who are proficient in Chinese (although we plan to address this issue in the future).
```

Reply:
Thanks for the comment, I will modify it.